lisa/sut_orchestrator/azure/platform_.py

```diff
@@ -1281,6 +1294,8 @@ def _create_node_runbook(
                 f"but {azure_node_runbook.hyperv_generation}",
             )
         if azure_node_runbook.vhd and azure_node_runbook.vhd.vhd_path:
+            node_context = get_node_context(node)
+            node_context.original_vhd_path = azure_node_runbook.vhd.vhd_path
```

Comment:
I think it just needs to call `secreat.add_secret` here. So original vhd path will be automatically replaced to original in log.

Reply:
In this PR, I want to fix the error message in the database. The original vhd path is already in the secret list. But the secret just masks the token in the log, it doesn't mask the error message updated in the database. So we need to truly replace it in the error message.

We might have some misunderstanding. Could you review the description of this PR at the top of this page, let me know if you have any concerns.

Reply:
Thank you for bring it up! Understood it more.

1. Make sure the masked vhd path still be in log at least once, so we know where the vhd is copied to. We need to log it with existing mask, and then replace the pattern of secret to print original vhd path in future.
2. Use secret lib to filter exceptions, so no secret leak into error message as well.

Reply:
Done